framework/jobs/src/main/java/org/apache/cloudstack/framework/jobs/dao/AsyncJobDaoImpl.java

```diff
@@ -95,6 +100,26 @@ public AsyncJobDaoImpl() {
 
     }
 
+    @Override
+    public AsyncJobVO findById(Long id) {
```

Comment:
The findByIdIncluding removed previous usage was okay.

Reply:
reverted